cal/cal/static/css/calendaranalytics.scss

```diff
@@ -25,6 +25,12 @@ button {
   padding: 0 5 5 5;
 }
 
+.textbox input {
```

Comment:
Done. My initial issue was that changing the class to be placed on `input` also affected the checkbox input. Changed to `input[type=text]`!